signer/signer.go

```diff
@@ -55,15 +55,31 @@ type RSACacheConfig struct {
 
 // Configuration defines the parameters of a signer
 type Configuration struct {
-	ID                      string         `json:"id"`
-	Type                    string         `json:"type"`
-	Mode                    string         `json:"mode"`
-	PrivateKey              string         `json:"privatekey,omitempty"`
-	PublicKey               string         `json:"publickey,omitempty"`
-	Certificate             string         `json:"certificate,omitempty"`
-	X5U                     string         `json:"x5u,omitempty"`
-	RSACacheConfig          RSACacheConfig `json:"rsacacheconfig,omitempty"`
-	NoPKCS7SignedAttributes bool           `json:"nopkcs7signedattributes,omitempty"`
+	ID          string `json:"id"`
+	Type        string `json:"type"`
+	Mode        string `json:"mode"`
+	PrivateKey  string `json:"privatekey,omitempty"`
+	PublicKey   string `json:"publickey,omitempty"`
+	Certificate string `json:"certificate,omitempty"`
```

Comment:
nit: gofmt

Reply:
webui quirk? the `string` of `Certificate` is misaligned: https://screenshots.firefox.com/lz6CVo8LSu2Rb7GH/github.com

Reply:
oh that's bizarre I'm seeing : https://screenshots.firefox.com/aPYW8lY8SKavqYSE/github.com